react/ProductSummaryImage.tsx

```diff
@@ -183,12 +192,19 @@ function BadgeWrapper({
 
 function findImageByLabel(
   images: ProductSummaryTypes.SKU['images'],
-  selectedLabel: string | undefined
+  selectedLabel: string | undefined,
+  labelMatchCriteria: ImageLabelMatchCriteria | undefined = 'exact'
```

Comment:
The `undefined` isn't necessary.
```suggestion
  labelMatchCriteria: ImageLabelMatchCriteria = 'exact'
```

Reply:
@igorbrasileiro Makes sense, thank you 😄